health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/Attendance.js

```
@@ -0,0 +1,339 @@
+    {
+      name: t("REGISTER_CONTACT_NUMBER"),
+      selector: (row) => {
+        return (
+          <div className="ellipsis-cell" title={row?.mobileNumber || t("REGISTER_NA")}>
+            {row?.mobileNumber || t("NA")}
+          </div>
+        );
+      },
+      sortable: true,
+      sortFunction: (rowA, rowB) => {
+        const numberA = parseInt(rowA.number, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)